confident_metrics/__init__.py

```diff
@@ -1,3 +1,7 @@
 """Python abstraction for runtime metrics collection built on top of Prometheus."""
+try:
+    from confident_metrics.metrics import record_event  # noqa: F401
```

Comment:
This shall always be importable, why do we insert try-catch?

Reply:
The first time you install the package it loads the __version__ number from __init__.py, and it raises an ImportError because the dependencies are not installed yet.

Reply:
Got it. Then move `__version__` to `version.py`, import it here and in `setup.py`

Reply:
No try-catch in `__init__`

Reply:
the try catch was already removed, now I fixed the style also.